neptune/codelist.go

```diff
@@ -157,7 +169,21 @@ including the case of a short-circuit early termination of the query, because no
 list exists. It returns a wrapped error if a Code is found that does not have a "value" property.
 */
 func (n *NeptuneDB) GetCodes(ctx context.Context, codeListID, edition string) (*models.CodeResults, error) {
-	qry := fmt.Sprintf(query.GetCodes, codeListID, edition)
+
+	// Check if order is defined
+	qry := fmt.Sprintf(query.CountOrderedEdges, codeListID, edition)
+	orderedCount, err := n.getNumber(qry)
```

Comment:
Not sure if you have looked much into the performance here, but if we do see slow queries consider checking for the existence of a single edge with the order property, instead of finding them all for the count.

Reply:
good point. Another possible performance improvement would be to use the 'order' as pagination offset, maybe in another task?